decision_maker.go

```diff
@@ -225,31 +196,30 @@ func inspectUpgradeScenario(r *release, rs releaseState) {
 			// upgrade
 			diffRelease(r)
 			upgradeRelease(r)
-			logDecision(generateDecisionMessage(r, "release [ "+r.Name+" ] is desired to be upgraded. Planning this for you!", false), r.Priority, change)
+			logDecision("release [ "+r.Name+" ] will be upgraded.", r.Priority, change)
 
 		} else if extractChartName(r.Chart) != getReleaseChartName(rs) {
 			reInstallRelease(r, rs)
-			logDecision(generateDecisionMessage(r, "release [ "+r.Name+" ] is desired to use a new Chart [ "+r.Chart+
-				" ]. I am planning a purge delete of the current release and will install it with the new chart in namespace [[ "+
-				r.Namespace+" ]]", false), r.Priority, change)
+			logDecision("release [ "+r.Name+" ] is desired to use a new Chart [ "+r.Chart+
+				" ]. Delete of the current release will be planned and new chart will be installed in namespace [[ "+
+				r.Namespace+" ]]", r.Priority, change)
 		} else {
 			if diff := diffRelease(r); diff != "" {
 				upgradeRelease(r)
-				logDecision(generateDecisionMessage(r, "release [ "+r.Name+" ] is currently enabled and have some changed parameters. "+
-					"I will upgrade it!", false), r.Priority, change)
+				logDecision("release [ "+r.Name+" ] is installed and has some changes to apply. "+
+					"I will upgrade it!", r.Priority, change)
```

Comment:
I would remove `I will upgrade it!` from the log to be consistent with the refactor.

Reply:
:+1: